acceptancetests/assess_recovery.py

```diff
@@ -133,14 +142,12 @@ def assess_recovery(bs_manager, strategy, charm_series):
     if strategy == 'ha-backup':
         enable_ha(bs_manager, controller_client)
         backup_file = controller_client.backup()
-        # at the moment we can't currently restore a backup in HA without
-        # removing the controllers and replica set. This test will need to
-        # be fix to accommodate this setup.
+        disable_ha(bs_manager, controller_client)
```

Comment:
this would then turn into 

    bs_manager.remote_machines = disable_ha(contrller_client)

Though we'd have to check for the failure cases (timeout/other failure to remove the HA units)